grammar.js

```diff
@@ -12,7 +12,7 @@ const PREC = {
   NEG: 9,
   INC: 10,
   NON_NULL: 10,
-  FUNCTION_CALL: 12,
+  FUNCTION_CALL: 11,
```

Comment:
👍 matches [the precedence change](https://github.com/tree-sitter/tree-sitter-javascript/pull/89/files#diff-95fc33185b7215010b60f43469000a7aR21).